aptos-move/aptos-vm-types/src/output.rs

```diff
@@ -135,13 +146,6 @@ impl VMOutput {
                 .is_empty(),
             "Delayed fields must be empty after materialization."
         );
-        assert!(
```

Comment:
I guess there is no longer an easy way to check resource groups have been combined? can/do we check instead (here or in extend_resource_write_set, maybe in debug mode only) that all required abstract ops have been patched? That seems like a super useful check potentially (now we may even make it a PanicError it seems?)